### PR TITLE
fixes some warnings

### PR DESCRIPTION
**Pull request by niteshcode4u**
This is the changes for basic warning fixes and to test the first commit

### CODE REVIEW

**Comment by virusnitesh** [comment]
Will add more on this please verify this is the process we are following @arjun289 @Ayoush @BandanaPandey

File: config/dev.exs [hunk 1]
```diff
@@ -68,6 +68,7 @@ config :acqdat_core, AcqdatCore.Repo,
   password: "postgres",
   database: "acqdat_core_dev",
   hostname: "localhost",
+  port: 5431,
```
Comment (oyeb) [inline config/dev.exs:71]: Perhaps `port: System.get_env("LOCAL_DB_PORT", "5432")` to avoid breaking others' setup.

**Review by oyeb — CHANGES_REQUESTED** [review]

File: config/dev.exs (hunk 1 above)
Comment (oyeb) [reply config/dev.exs:71]: nvm, Arjun made this 5431 in #252.

**Comment by oyeb** [comment]
@arjun289 The build for this PR passed. Is the ElasticSearch test failure sporadic? Or has it been fixed in this PR?

**Pull request merged**